config.cson

```diff
@@ -2953,6 +2953,7 @@ fileIcons:
 			[".huskyrc.json", "auto-yellow"]
 			[".huskyrc.cjs", "auto-green"]
 			[".huskyrc.mjs", "auto-blue"]
+			[".huskyrc.yml", "auto-red"]
```

Comment:
I'd change `".huskyrc.yml"` to `/\.huskyrc\.ya?ml$/i` instead, so that it matches both `.yml` and `.yaml` extensions. To quote @typicode's [response](https://github.com/typicode/husky/issues/367#issuecomment-426227759) in the aforementioned thread:

> It was just to simplify docs, but `yml` will be supported :) (actually as long as `cosmiconfig` supports it out of the box)

... and both `yaml` and `yml` are listed in CosmiConfig's [documented defaults](https://www.npmjs.com/package/cosmiconfig).

Reply:
Good point, changed it 👍